### ISSUE TITLE
[BUG] 12.4.0 default SVG format gives a light background on syntax lines

### ISSUE THREAD

**Issue by paw-lu** [Textualize/rich#2260]
**Describe the bug**

### Summary

Exporting content created by `rich.syntax.Syntax` to SVG using the new default template added in 12.4.0 results in a light highlight over the resulting syntax. This is also present when `background_color` is not `"default"`, but it's a little more obvious when it is.

This issue was not present in the previous template.

### Minimal

```python
from rich import console, syntax

print_syntax = syntax.Syntax(
    'print("hey")',
    "python",
    background_color="default",
)
svg_console = console.Console(
    width=79,
    record=True,
    color_system="truecolor",
    legacy_windows=False,
    force_terminal=True,
    force_interactive=True,
)
svg_console.print(print_syntax)
svg_console.save_html("debug.html", clear=False)
svg_console.save_svg("debug.svg", clear=False)
```

### Output
Screenshot on Chrome 101

#### SVG

![debug](https://user-images.githubusercontent.com/30049606/167273971-b14c3400-8c85-400c-8c8a-76866682eaf4.svg)

#### Screenshot

<img width="880" alt="Screen Shot 2022-05-07 at 3 30 51 PM" src="https://user-images.githubusercontent.com/30049606/167273938-613a8076-08a2-44cc-ba0d-fefe36b32565.png">

**Platform**
<details>
<summary>Click to expand</summary>

```txt
╭───────────────────────── <class 'rich.console.Console'> ─────────────────────────╮
│ A high level console interface.                                                  │
│                                                                                  │
│ ╭──────────────────────────────────────────────────────────────────────────────╮ │
│ │ <console width=250 ColorSystem.TRUECOLOR>                                    │ │
│ ╰──────────────────────────────────────────────────────────────────────────────╯ │
│                                                                                  │
│     color_system = 'truecolor'                                                   │
│         encoding = 'utf-8'                                                       │
│             file = <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'> │
│           height = 69                                                            │
│    is_alt_screen = False                                                         │
│ is_dumb_terminal = False                                                         │
│   is_interactive = True                                                          │
│       is_jupyter = False                                                         │
│      is_terminal = True                                                          │
│   legacy_windows = False                                                         │
│         no_color = False                                                         │
│          options = ConsoleOptions(                                               │
│                        size=ConsoleDimensions(width=250, height=69),             │
│                        legacy_windows=False,                                     │
│                        min_width=1,                                              │
│                        max_width=250,                                            │
│                        is_terminal=True,                                         │
│                        encoding='utf-8',                                         │
│                        max_height=69,                                            │
│                        justify=None,                                             │
│                        overflow=None,                                            │
│                        no_wrap=False,                                            │
│                        highlight=None,                                           │
│                        markup=None,                                              │
│                        height=None                                               │
│                    )                                                             │
│            quiet = False                                                         │
│           record = False                                                         │
│         safe_box = True                                                          │
│             size = ConsoleDimensions(width=250, height=69)                       │
│        soft_wrap = False                                                         │
│           stderr = False                                                         │
│            style = None                                                          │
│         tab_size = 8                                                             │
│            width = 250                                                           │
╰──────────────────────────────────────────────────────────────────────────────────╯
╭─── <class 'rich._windows.WindowsConsoleFeatures'> ────╮
│ Windows features available.                           │
│                                                       │
│ ╭───────────────────────────────────────────────────╮ │
│ │ WindowsConsoleFeatures(vt=False, truecolor=False) │ │
│ ╰───────────────────────────────────────────────────╯ │
│                                                       │
│ truecolor = False                                     │
│        vt = False                                     │
╰───────────────────────────────────────────────────────╯
╭──────────────────────────────────────────────────────────────────────────────────────────── Environment Variables ────────────────────────────────────────────────────────────────────────────────────────────╮
│ {'TERM': 'xterm-256color', 'COLORTERM': 'truecolor', 'CLICOLOR': None, 'NO_COLOR': None, 'TERM_PROGRAM': 'iTerm.app', 'COLUMNS': None, 'LINES': None, 'JPY_PARENT_PID': None, 'VSCODE_VERBOSE_LOGGING': None} │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
platform="Darwin"
```

</details>

---

Really enjoying this feature by the way, thanks for all the work!

**Comment by willmcgugan** [comment]
The default background rendering light was a bug, but not sure what you mean about the `highlight present when background_color is not "default"`. I can't reproduce that. Can you add a screenshot?

**Comment by paw-lu** [comment]
> but not sure what you mean about the highlight present when background_color is not "default"

Mistake on my end! Bug only appears when `background_color="default"`. Sorry for the confusion caused there.

**Comment by willmcgugan** [comment]
No worries. Will be fixed in 12.4.1 :)

**Comment by paw-lu** [comment]
Tested out 12.4.1, and it seems fixed for me! So I'm going to close out this issue!

Thanks a bunch!
